csrankings.csv

```diff
@@ -5211,7 +5211,7 @@ Eleanor O'Rourke,Northwestern University,http://www.eleanorourke.com,DRt16WsAAAA
 Eleazar Eskin,University of California - Los Angeles,http://www.cs.ucla.edu/~eeskin,VBXG4CwAAAAJ
 Eleftherios Garyfallidis,Indiana University,https://grg.sice.indiana.edu,Ln2EyRYAAAAJ
 Eleftherios Kokoris-Kogias,IST Austria,https://ist.ac.at/en/research/kokoris-group,7TOIlqkAAAAJ
-Elena Agapie,University of California - Irvine,https://eagapie.com,nc3IT3kAAAAJ
+Elena Agapie,University of California - Irvine,https://www.informatics.uci.edu/explore/faculty-profiles/elena-agapie/,nc3IT3kAAAAJ

```

Comment:
This and the other updated home page should be reverted; this is the actual faculty home page, and the institutional one is not.

Reply:
I changed them all to the faculty's personal homepage as opposed to the institutional ones.